### PR TITLE
Create Tables for Model Enums and Populate Table with Known Data

### LINKED ISSUE #30: Move Enums into SQL Tables

### PR DESCRIPTION
**Pull request by david-hu23**
closes #30

### CODE REVIEW

File: analyzer-sql/src/main/liquibase/ffb/analyzer/sql/data/player-status.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,value,label
+0,ON_TEAM,On Team
```
Comment (pecjas) [inline analyzer-sql/src/main/liquibase/ffb/analyzer/sql/data/player-status.csv:2]: Just confirmation - sinnce we're converting ON_TEAM and ONTEAM to one enum value when interpreting ESPNs data, we shouldn't need to handle both in the database, correct?

File: analyzer-sql/src/main/liquibase/ffb/analyzer/sql/data/position.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+id,value,label
+1,QB,Quarterback
+2,RB,Running Back
+3,WR,Wide Receiver
+4,TE,Tight End
+5,K,Kicker
+16,DEFENSE,Defense
```
Comment (pecjas) [inline analyzer-sql/src/main/liquibase/ffb/analyzer/sql/data/position.csv:7]: Looking at our dummy league, there are a lot of other positions we could add to this. We should do a draft soon with all of the possible positions enabled so we can see the position id's for each. Or there may be an ESPN API that provides this information.

File: analyzer-sql/src/main/liquibase/ffb/analyzer/sql/data/player-status.csv (hunk 1 above)
Comment (david-hu23) [reply analyzer-sql/src/main/liquibase/ffb/analyzer/sql/data/player-status.csv:2]: Correct.

File: analyzer-sql/src/main/liquibase/ffb/analyzer/sql/data/position.csv (hunk 2 above)
Comment (pecjas) [reply analyzer-sql/src/main/liquibase/ffb/analyzer/sql/data/position.csv:7]: Created Issue 45 to track adding the remainder of the positions.

**Pull request merged**